util_test.go

```diff
@@ -151,11 +174,13 @@ func (matcher *singleTransactionMatcher) FailureMessage(actual interface{}) (mes
 				transactions[first],
 			)
 		} else {
-			// TODO: maybe show transaction that is closest to matching?
-			return fmt.Sprintf("Expected the sequence of transactions to include: %T. %d of the same type were found, but none of them matched.",
-				matcher.expected,
-				count,
-			)
+			expected := ""
+			closest := getClosestTransaction(transactions, matcher.expected)
+			for name, val := range closest {
+				expected += fmt.Sprintf("- %s\n%s\n", name, val)
+			}
+			return fmt.Sprintf("Closest %T (from %d total) has %d fields that do not match:\n%s",
```

Comment:
would prefer "x/y fields that do not match" or "x out of y fields that do not match"